.github/workflows/main.yml

```diff
@@ -61,3 +61,36 @@ jobs:
       - name: Build
         run: CI=false yarn build
         working-directory: examples/${{ matrix.example }}
+
+  deploy_docs:
```

Comment:
So this job is going to run on every PR and push, right? 

Is there any way to run a versioned Docusaurus page on GitHub Pages? So that we can keep documentation for different releases up. Alternatively, we could only run `deploy_docs` on the latest tag (latest release). What do you think?

Reply:
yes there is https://docusaurus.io/docs/versioning i just wanted to get a basic workflow going for now (although i'm hitting permission errors on the `nucypher` repo that i didn't get on my own

Reply:
Sounds good. I also only ever published the `latest` version of docs on GH.